src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.Guid.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text
+{
+    public static partial class PrimitiveFormatter
+    {
+        public static bool TryFormat(this Guid value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), TextEncoder encoder = null)
+        {
+            encoder = encoder == null ? TextEncoder.Utf8 : encoder;
+
+            if (encoder.IsInvariantUtf8)
+                return InvariantUtf8GuidFormatter.TryFormat(value, buffer, out bytesWritten, format);
+            else if (encoder.IsInvariantUtf16)
+                return InvariantUtf16GuidFormatter.TryFormat(value, buffer, out bytesWritten, format);
+            else
+                throw new NotImplementedException();
```

Comment:
This should already be covered by issue #57. There are variants of that issue for parsing and encoding as well.